src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -701,117 +701,252 @@ int wm_vuldet_send_agent_report(sqlite3 *db, OSHash *cve_table, agent_software *
+                    cvss_ver = (char *)sqlite3_column_text(stmt, 4);
+
+                    if (!cvss_ver) {
+                        continue;
+                    }
 
-                int bug_references = 0;
-                while (sql_result = wm_vuldet_step(stmt), sql_result == SQLITE_ROW) {
-                    bugzilla_reference = (char *)sqlite3_column_text(stmt, 0);
-                    os_realloc(report->bugzilla_references, (bug_references + 2) * sizeof(char *), report->bugzilla_references);
-                    os_strdup(bugzilla_reference, report->bugzilla_references[bug_references]);
-                    report->bugzilla_references[++bug_references] = NULL;
+                    if (!strcmp(cvss_ver, "2.0") && !report->cvss2) {
+                        cvss = &report->cvss2;
+                    } else if (!strcmp(cvss_ver, "3.0") && !report->cvss3) {
+                        cvss = &report->cvss3;
+                    } else {
+                        continue;
+                    }
+
+                    os_calloc(1, sizeof(cv_scoring_system), *cvss);
+                    w_strdup(vector_string, (*cvss)->vector_string);
+                    (*cvss)->base_score = base_score;
+                    (*cvss)->exploitability_score = exploitability_score;
+                    (*cvss)->impact_score = impact_score;
+                }
+                sqlite3_finalize(stmt);
+
+                if (report->cvss3) {
+                    if (report->cvss3->base_score >= 9) {
+                        os_strdup(vu_severities[VU_CRITICAL], report->severity);
+                    } else if (report->cvss3->base_score >= 7) {
+                        os_strdup(vu_severities[VU_HIGH], report->severity);
+                    } else if (report->cvss3->base_score >= 4) {
+                        os_strdup(vu_severities[VU_MEDIUM], report->severity);
+                    } else if (report->cvss3->base_score >= 0.1) {
+                        os_strdup(vu_severities[VU_LOW], report->severity);
+                    } else {
+                        os_strdup(vu_severities[VU_NONE], report->severity);
+                    }
+                } else if (report->cvss2) {
+                    if (report->cvss2->base_score >= 7) {
+                        os_strdup(vu_severities[VU_HIGH], report->severity);
+                    } else if (report->cvss2->base_score >= 4) {
+                        os_strdup(vu_severities[VU_MEDIUM], report->severity);
+                    } else {
+                        os_strdup(vu_severities[VU_LOW], report->severity);
+                    }
+                } else {
+                    os_strdup(vu_severities[VU_UNKNOWN], report->severity);
                 }
 
-                // Adding advisories
-                if (sqlite3_prepare_v2(db, vu_queries[VU_ADVISORY_QUERY], -1, &stmt, NULL) != SQLITE_OK) {
+                //os_strdup(report_node->generated_cpe, report->generated_cpe);
```

Comment:
I would not add this field to the Linux alerts since we are not using the CPE to detect the vulnerabilities. Can you remove the comment?

Reply:
DONE